src/lib.rs

```diff
@@ -270,18 +306,12 @@ enum Field {
     Ascii,
 }
 
-//TODO: Maybe make them configurable anyhow. Ideas?!
-const U8S_PER_LINE: usize = 16;
-const NIBBLES_PER_LINE: usize = U8S_PER_LINE * 2;
-const CHARS_PER_GROUP: usize = 4;
-const CHARS_PER_SPACING: usize = 1;
-const ADDR_HEX_SEPERATOR: &str = ": ";
-const HEX_ASCII_SEPERATOR: &str = " │ ";
+
 
 /// calcs the position in a line with spacing
-fn get_cursor_offset(vec: Vec2) -> Vec2 {
+fn get_cursor_offset(vec: Vec2, config: HexViewConfig) -> Vec2 {
```

Comment:
why is this accepted as value instead of reference? We don't need a copy of it, just pass a `&HexViewConfig` and you can omit a lot of `clone`s in the code

Reply:
Let me know if [this](https://github.com/palako/cursive_hexview/commit/c31ca2e847910ce810c2c4b71ccbb5d274f7945a) is good